src/protocol/malicious.rs

```diff
@@ -115,19 +112,11 @@ impl<F: Field> MaliciousValidatorAccumulator<F> {
 
     /// ## Panics
     /// Will panic if the mutex is poisoned
-    pub fn accumulate_macs(
-        &self,
-        prss: &Arc<IndexedSharedRandomness>,
-        record_id: RecordId,
-        input: &MaliciousReplicated<F>,
-    ) {
+    pub fn accumulate_macs(&self, random_constant: &Replicated<F>, input: &MaliciousReplicated<F>) {
```

Comment:
Wait a second... where did `record_id` go? How does this work without record_id?